src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/socialEmbed/index.test.js

```diff
@@ -0,0 +1,65 @@
+import socialEmbed from '.';
+import pageData from './fixtures';
+
+describe('socialEmbed', () => {
+  const { blocks } = pageData.content;
+  const twitter = blocks[1];
+  const twitterNoEmbed = blocks[3];
+
+  it('should convert a social_embed block to Optimo format', () => {
```

Comment:
Is this test name a bit misleading?

Reply:
Hmm, what would you suggest I rename it to? Here we're testing whether the data we receive is converted into the correct format. The directory this function resides in is named "convertToOptimoBlocks".

Reply:
If it’s consistent with the naming of comparable tests elsewhere it’s not a big deal.